### ISSUE TITLE
Sphinx-1.5 release plan

### ISSUE THREAD

**Issue by tk0miya** [sphinx-doc/sphinx#2872]
Hi developers,

I'm planning to release Sphinx-1.5 (alpha|beta|final) as below.
- Sphinx-1.5a1: middle of Sept.
- Sphinx-1.5a2: end of Sept.
- Sphinx-1.5b1: middle of Oct.
- Sphinx-1.5-final: end of Oct. or middle of Nov.

Alpha is: Evaluation period for new features. We will have a2 and a3 if it is needed.
Beta is: Feature freeze. Call for translation

Nonetheless, many tickets are still exist for version 1.5.
https://github.com/sphinx-doc/sphinx/milestones/1.5
If it didn’t meet the deadline, I'll postpone the non-closed issues/PRs to next alpha version or next major version.

Your comments, suggestions and feedback are always welcome.
https://groups.google.com/forum/#!topic/sphinx-dev/QikqSL0anqo

Regards,

**Comment by tk0miya** [comment]
It't time to 1.5a2, but unfortunately there are no updates from 1.5a1.
So I determined to postpone or cancel 1.5a2 release.

At this moment, I can't say the effect to due date of beta and final release.
But we still have many issues for 1.5, so we have to discuss about the date.
Thanks,

**Comment by tk0miya** [comment]
This weekend is due date for beta release at first.
But we still have 38 issues for v1.5.
In addition, I don't have enough time for Sphinx since Sept. (maybe, to end of Oct).

So I propose a reschedule of the plan.
IMO, it is needed postponement of a month.
- Sphinx-1.5a2: end of Oct.
- Sphinx-1.5b1: middle of Nov.
- Sphinx-1.5-final: end of Nov. or middle of Dec.

At same time, we should move some issues to v1.6. Especially, I could not check the issues/PRs for autodoc.

Any comments?

**Comment by tk0miya** [comment]
Today, I and @shimizukawa discussed about release plan for 1.5.

As a result, we rescheduled as below:
- Sphinx-1.5a2: in this weekend (Oct. 16)
- Sphinx-1.5b1: Nov. 5th or 6th
- Sphinx-1.5-final: end of Nov.

So all our work in develop should finished until Nov. 5th. After that, no new features should not be implemented. Only bug fixes are allowed.

Thank you always for everything

**Comment by tk0miya** [comment]
@jfbu Be careful to the date for feature freeze.

**Comment by tk0miya** [comment]
Hi developers,
I will release 1.5b1 in this weekend; Nov 6th (maybe at 18-20 pm JST, 9-11 am UTC).
Do you have any issues that does not end until the due date?
If so, please let me know.

After the release, feature freeze will start. All we should not introduce any new features to Sphinx-1.5. Only bug fixes are allowed.

**Comment by tk0miya** [comment]
@shimizukawa @jfbu 
Please let me know if you need any helps, advices, reviews and so on.
I will work to resolve them with my best.
Thank you always, and thank you for your support!

**Comment by shimizukawa** [comment]
Woo Hoo!

**Comment by jfbu** [comment]
exciting times ;-) I will check tomorrow afternoon (European time) Nov 3rd the status of open issues and PRs related to LaTeX ...

**Comment by tk0miya** [comment]
I will release 1.5b1 later (maybe at 2-3 hours later).

**Comment by jfbu** [comment]
good ;-) : I profited to push typo fixing commit ...

**Comment by jfbu** [comment]
this may not the best place to report, but whenever I build the latex docs I get (this is not new):

```
writing... /xxx/sphinx/doc/faq.rst:: WARNING: unusable reference target found: info:Texinfo#makeinfo_options
done
```

**Comment by tk0miya** [comment]
Thanks, fixed at https://github.com/sphinx-doc/sphinx/commit/4250052eda05717d8069b0c4820d3978c3760fe5

**Comment by jfbu** [comment]
I edited again at c94f3b7, because output looked weird (two identical code samples).

![capture d ecran 2016-11-06 a 12 53 40](https://cloud.githubusercontent.com/assets/2589111/20037509/bb30a030-a420-11e6-97ad-72bdfb8c322f.png)

**Comment by tk0miya** [comment]
Okay, it's time to release. Do you have any task for 1.5b1?

**Comment by tk0miya** [comment]
@jfbu Do you have any task for 1.5b1?

**Comment by jfbu** [comment]
@tk0miya there is one thing, I just realized that in inline literals, LaTeX will gobble a space if it comes after `-`. This is not new. I can fix it immediately or wait.

**Comment by tk0miya** [comment]
Okay. Please let me know after your work finished.
I will start packaging.
It's not hurry. Do it at your own pace :-)

**Comment by jfbu** [comment]
thanks for patience @tk0miya. I am done now, pushed commit.

**Comment by tk0miya** [comment]
@jfbu Thanks!

I will start packaging after travis says ok.
https://travis-ci.org/sphinx-doc/sphinx/builds/173660250

**Comment by tk0miya** [comment]
Hi developers,

I just released 1.5b1.
From now on, we use "1.5-release" branch to fix bugs only.
Please add new features to master branch (It will be released as 1.6).

Thank you for all your support!

**Comment by shimizukawa** [comment]
Congratulation!! 🎉  Thanks for your contribution and leading!

**Comment by tk0miya** [comment]
Hi developers,
It seems no bugs are reported for 1.5b1.
So I will release this weekend.
What do you think? If you have critical or important bugs, please let me know.

Thanks,

**Comment by shimizukawa** [comment]
Please refer #3182

**Comment by tk0miya** [comment]
Okay, I'll take a look.
Unfortunately I'm very busy in this weekdays. So I will check it in this weekend.

**Comment by jfbu** [comment]
Can we merge #3180 into 1.5-release ?

**Comment by tk0miya** [comment]
@jfbu As a quick look, it seems #3180 is important change for 1.5 to support `is_multiline`. Please merge it into 1.5-release.
It's okay to merge because it is already reviewed by jakobandersen :-)

**Comment by jfbu** [comment]
thanks @tk0miya @jakobandersen I have merged it at d2df39d.

**Comment by tk0miya** [comment]
Hi developers,
It's time to release of Sphinx-1.5. I and @shimizukawa will start packaging within 12-24 hours.
If you still have work for 1.5, please let us know that and finish ASAP.

Thanks,

**Comment by shimizukawa** [comment]
Now start packaging

**Comment by shimizukawa** [comment]
published https://pypi.python.org/pypi/Sphinx/1.5

**Comment by shimizukawa** [comment]
ToDo:

* send announce mail
* publish translated document in rtd.

**Comment by tk0miya** [comment]
@shimizukawa Do you still have any work for 1.5-release?
If finished, please close this issue.

**Comment by shimizukawa** [comment]
Ah, all works has been done!
